src/app/core/services/rundown-timing-context-state.service.ts

```diff
@@ -97,7 +98,7 @@ export class RundownTimingContextStateService {
   }
 
   private hasActiveRundown(): boolean {
-    return [...this.rundowns.values()].some(rundown => rundown.isActive)
+    return [...this.rundowns.values()].some(rundown => rundown.mode === RundownMode.ACTIVE)
```

Comment:
Since this is used to determine if we should have a high or low time resolution for recalculating the timings, it would be more appropriate to have this check if there are any rundowns not inactive.
```typescript
    return [...this.rundowns.values()].some(rundown => rundown.mode !== RundownMode.INACTIVE)
```

Reply:
But the method is called `hasActiveRundown`. The method implementation doesn't know what it is being used for. And it could easily be used for more than what it is currently being used for.
That it is being used to determine the time resolution for recalculating timings is out of scope of the method.

So either we do:
```
private hasActiveOrRehearsalRundown(): boolean {
  return [...this.rundowns.values()].some(rundown => [RundownMode.ACTIVE, RundownMode.REHEARSAL].includes(rundown.mode))
}
```
Or 
```
private isAllRundownsInactive(): boolean {
  return [...this.rundowns.values()].every(rundown => rundown.mode === RundownMode.INACTIVE)
}
```

I have opted for the last suggestion and switched the values returned in the ternary where the method is used.